### PR TITLE
Update anydesk.rb

### PR DESCRIPTION
**Pull request by silverbios**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by core-code** [comment]
do you also have an 'appcast' for this one?

File: Casks/anydesk.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "anydesk" do
-  version :latest
-  sha256 :no_check
+  version :5.6.0
```
Comment (vitorgalvao) [inline Casks/anydesk.rb:2]: `audit` wouldn’t pass this.

**Comment by miccal** [comment]
[This may make a suitable `appcast`.](https://anydesk.com/en/downloads/mac-os)

**Review by vitorgalvao — APPROVED** [review]

**Comment by stonerl** [comment]
Why has this one been merged? According to the documentation : `:latest`should be used, because there is no versioned download link.

**Comment by vitorgalvao** [comment]
> Why has this one been merged?

Because it has an `appcast`, which makes it possible to track versions without the cask constantly breaking.

**Comment by stonerl** [comment]
Hmm, I don't get this. Does this mean the cask gets automatically updated from now on?

**Pull request merged**